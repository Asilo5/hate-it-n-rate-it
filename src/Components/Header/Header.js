import React from 'react';
import './Header.scss'

const Header = () => {
  return (
    <div className='header'>
    <h1><span>Hate</span> It 'N <span>Rate</span> It</h1>
    </div>
  )
}

export default Header;