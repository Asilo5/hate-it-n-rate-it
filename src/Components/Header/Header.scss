.header {
  height: 8vh;
  span {
    color: purple;
  }
}