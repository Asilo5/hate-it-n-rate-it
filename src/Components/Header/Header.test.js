import React from 'react';
import { shallow } from 'enzyme';
import { Header } from './Header';

describe('header', () => {
  let wrapper;
  let mockUser = {};

  beforeEach(() => {
    wrapper = shallow(<Header user={mockUser} />)
  })

  describe('snapshots', () => {
    it('should match the snapshot', () => {
      expect(wrapper).toMatchSnapshot();
    })

    it('should match snapshot without user', () => {
      wrapper = shallow(<Header user={{}} />);
      expect(wrapper).toMatchSnapshot();
    })
  })
})