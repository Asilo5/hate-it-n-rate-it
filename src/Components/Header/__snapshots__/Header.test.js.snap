// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header snapshots should match snapshot without user 1`] = `
<header>
  <h1>
    Hate It 'N Rate It
  </h1>
  <div>
    <Connect(NavBar) />
  </div>
  <img
    alt="pattern decoration"
    src="http://www.thecareeravenue.com/Images/extras/spacer.png"
  />
</header>
`;

exports[`header snapshots should match the snapshot 1`] = `
<header>
  <h1>
    Hate It 'N Rate It
  </h1>
  <div>
    <Connect(NavBar) />
  </div>
  <img
    alt="pattern decoration"
    src="http://www.thecareeravenue.com/Images/extras/spacer.png"
  />
</header>
`;
